opentelemetry-sdk/tests/metrics/test_view.py

```diff
@@ -37,6 +37,15 @@ def test_instrument_name(self):
             View(instrument_name="instrument_name")._match(mock_instrument)
         )
 
+    def test_instrument_unit(self):
+
+        mock_instrument = Mock()
+        mock_instrument.configure_mock(**{"unit": "instrument_unit"})
```

Comment:
nit: looks like you can do this more succinctly with:

```
mock_instrument.unit = 'instrument_unit'
```

but `configure_mock` is used elsewhere so 🤷

Reply:
Thank you for the comment, It seems that you are right, but I prefer not to change the way all the tests work